src/react-components/src/components/02-molecules/Cards/EditionCard.stories.tsx

```diff
@@ -47,6 +47,16 @@ export const editionCardWithSomeEditionInfo = () => <EditionCard
   downloadLink="#downloadUrl">
 </EditionCard>;
 
+export const editionCardWithComponentInfo = () => <EditionCard
+  id="card#1"
+  blockName=""
+  coverUrl="https://placeimg.com/300/400/arch"
+  editionHeadingElement={<a href="edition-link" className={bem("link", [], "heading")} >2004 Edition</a>}
+  editionInfo={[<span>Published in New York</span>, "Under Creative Commons License"]}
```

Comment:
1) Whatever is passed as an edition info will always be wrapped in a `<span>`. So now, for this example, the first info text will be wrapped in two spans and the second info text will be wrapped in one.

2) Also, I do understand that there will be documentation on how to use this component and what to pass as a prop, but that won't stop anyone from passing something like this which is correct html markup:
```
editionInfo={[<div><dl><dt>owl</dt><dd>a type of bird</dd></dl></div>, "Under Creative Commons License"]}
```
![Screen Shot 2020-02-20 at 2 35 26 PM](https://user-images.githubusercontent.com/1280564/74971559-4b5e1400-53ee-11ea-8e85-1fcfff3217f1.png)

This worries me as more props are now more lenient and can accept almost anything. But, what's the rationale for making these props allow an html element?

Reply:
So this change is driven by a need for "Under Creative Commons License" to be a link to a license explanations page.  I'm not sure if there's a more restrictive way to allow something like this, because links can be passed as `<a>` or as `<ReactRouter.Link>`.  And if we allow `<a>` should we allow `<em>` or `<strong>` or anything like that?  Should we allow `<span>` so that it can be styled with a class?  if we allow `<span>` how can we stop people from doing the thing you demonstrated?

I think it comes down to a larger question around what this component should be - is it a card that can only be used for editions of books?  Or is it a card that can display an image on the left, some text in the middle and a call to action on the right?  Or something in between?  How might UX want to repurpose something like this?  

I don't have answers for this - All I know is that when product says, "Can you turn those words into a link" and I go "yes, and this will take a day and require approval from the Design System team" ... that ... might be the kind of thing where, over a long time, be a crazily frustrating amount of friction?

Reply:
Yeah, that makes sense. You could potentially check that what was passed is either an anchor or a react Link component, but on the larger scheme of things, it seems too specific and a lot of logic for this component to handle. So if you will be lenient like this to make it easier for you later on, I think that's fine...

I think you are missing tests for the new prop values which can now accept an element besides a string, like the downloadLink and readOnlineLink.

I was going to say, you should also test `language` and `license` which can now be elements, but those values are not being rendered directly as props in the EditionCard. I now see that those values are being taken out of the `editionInfo` object, merged together, and passed together as `editionInfo` in the `EditionCard`. You should test the case this reason came up and pass in a link for the license.